CLI/Testing/Cxx/RegressionTests.cmake

```diff
@@ -0,0 +1,228 @@
+#-----------------------------------------------------------------------------
+# Regression Tests QINProstate001
+#-----------------------------------------------------------------------------
+set(inputDataBaseName ${CMAKE_CURRENT_SOURCE_DIR}/../../../Data/RegressionTests/QINProstate001/Input/QINProstate001)
+set(referenceDataBaseDir ${CMAKE_CURRENT_SOURCE_DIR}/../../../Data/RegressionTests/QINProstate001/Reference/)
+
+#-----------------------------------------------------------------------------
+set(testName QINProstate001_AllOutputsExceptFpv)
+set(tempOutDataBaseName ${TEMP}/${testName})
+add_test(NAME ${testName} COMMAND ${Launcher_Command} $<TARGET_FILE:${CLP}Test>
+  --compareIntensityTolerance 0.0
+  --compare ${referenceDataBaseDir}${testName}-conc.nrrd
+  ${tempOutDataBaseName}-conc.nrrd
```

Comment:
as far as I understand the source code of the itkTestingComparisonImageFilter the iterators should go through all pixels of the images :/